File: scope104/src/main/java/com/petro/scope104/util/DateFormatHelper.java
package com.petro.scope104.util;

import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;
import java.util.concurrent.TimeUnit;

public class DateFormatHelper {
    public static String formatDob(Date date){
        final SimpleDateFormat formatter = new SimpleDateFormat("dd MMMM yyyy", Locale.getDefault());
        int yearsCount = (int) (TimeUnit.MILLISECONDS.toDays(System.currentTimeMillis() - date.getTime())/365);
        return String.format(Locale.getDefault(),"%s, %d years", formatter.format(date), yearsCount);
    }
}

File: scope104/src/main/kotlin/com/petro/scope104/util/DateFormatHelper.kt
package com.petro.scope104.util

import java.text.SimpleDateFormat
import java.util.*
import java.util.concurrent.TimeUnit

object DateFormatHelper {
    fun formatDob(date: Date?): String {
        val formatter = SimpleDateFormat("dd MMMM yyyy", Locale.getDefault())
        val yearsCount =
            (TimeUnit.MILLISECONDS.toDays(System.currentTimeMillis() - date!!.time) / 365).toInt()
        return String.format(
            Locale.getDefault(),
            "%s, %d years",
            formatter.format(date),
            yearsCount
        )
    }
}